pkg/server/aggregator.go

```diff
@@ -48,10 +50,23 @@ func mergeConfiguration(configurations config.Configurations) config.Configurati
 			conf.TLSStores[key] = store
 		}
 
-		for key, config := range configuration.TLSOptions {
-			conf.TLSOptions[key] = config
+		for tlsOptionsName, config := range configuration.TLSOptions {
+			if tlsOptionsName != "default" {
+				tlsOptionsName = internal.MakeQualifiedName(provider, tlsOptionsName)
+			} else {
+				defaultTLSOptionProviders = append(defaultTLSOptionProviders, provider)
+			}
+
+			conf.TLSOptions[tlsOptionsName] = config
 		}
 	}
 
+	if len(defaultTLSOptionProviders) == 0 {
+		conf.TLSOptions["default"] = tls.TLS{}
+	} else if len(defaultTLSOptionProviders) > 1 {
+		log.WithoutContext().Errorf("Default TLS Options defined multiple times in %v", defaultTLSOptionProviders)
+		delete(conf.TLSOptions, "default")
```

Comment:
```suggestion
		// We do not set an empty tls.TLS{} as above so that we actually get a "cascading failure" later on,
		// i.e. routers depending on this missing TLS option will fail to initialize as well.
		delete(conf.TLSOptions, "default")
```